numbat/src/lib.rs

```diff
@@ -253,6 +256,106 @@ impl Context {
+                                .iter()
+                                .map(|(x, _)| x.as_str())
+                                .collect::<Vec<_>>()
+                                .join(", "),
+                        )
+                        + m::nl();
+                }
+
+                help += m::text("A unit of [") + md.readable_type + m::text("]") + m::nl();
+
+                if let Some(defining_info) = self.interpreter.get_defining_unit(&full_name) {
+                    let x = defining_info
+                        .iter()
+                        .filter(|u| !u.unit_id.is_base())
+                        .map(|unit_factor| unit_factor.unit_id.unit_and_factor())
+                        .next();
+
+                    if !prefix.is_none() {
+                        help += m::nl()
+                            + m::value("1 ")
+                            + m::type_identifier(keyword)
+                            + m::text(" = ")
+                            + m::value(prefix.factor().pretty_print())
+                            + m::space()
+                            + m::type_identifier(&full_name);
+                    }
+
+                    if let Some(BaseUnitAndFactor(prod, num)) = x {
+                        help += m::nl()
+                            + m::value("1 ")
+                            + m::type_identifier(&full_name)
+                            + m::text(" = ")
+                            + m::value(num.pretty_print())
+                            + m::space()
+                            + prod.pretty_print_with(|f| f.exponent, 'x', '/', true);
+                    } else {
+                        help +=
+                            m::nl() + m::type_identifier(&full_name) + m::text(" is a base unit");
+                    }
+                };
+
+                return help;
+            }
+        };
+
+        if let Some(l) = self.interpreter.lookup_global(keyword) {
+            let mut help = m::text("Variable: ") + m::identifier(keyword) + m::nl();
+            if let Some(name) = &l.metadata.name {
+                help += m::text(name) + m::nl();
+            }
+            if let Some(url) = &l.metadata.url {
+                help += m::string(url) + m::nl();
+            }
+            if l.metadata.aliases.len() > 1 {
+                help += m::text("Aliases: ")
+                    + m::text(
+                        l.metadata
+                            .aliases
+                            .iter()
+                            .map(|x| x.as_str())
+                            .collect::<Vec<_>>()
+                            .join(", "),
+                    )
+                    + m::nl();
+            }
+
+            if let Ok((_, results)) = self.interpret(keyword, CodeSource::Internal) {
+                help += m::nl() + results.to_markup(None, self.dimension_registry(), true);
+            }
+
+            return help;
+        }
+
+        m::text("Not found")
```

Comment:
It would be fantastic if we could also use this for types, i.e. do `info ElectricCharge` and get information about how it is defined and what the base representation is. But I guess we can leave this for another PR.